lexic.txt

```diff
@@ -1,37 +0,0 @@
-Alphabet:
-
-a. Upper (A-Z) and lower case letters (a-z) of the English alphabet
-b. Underline character '_';
-c. Decimal digits (0-9);
-
--operators + - * / < <= = >= eq mod ** not
-
-- separators [ ] { } ; space
-
-- reserved words: init verify otherwise escape true untrue while for func in
-
-integer:
-
-<int>::= 0 | <nonzero>
-<nonzero>::=<nonzerodigit>|-<nonzerodigit>|<nonzerodigit><digitseq>|-<nonzerodigit><digitseq>
```

Comment:
I would suggest separating concerns and also add a 
<non-zero-number> ::= <number> | +<number> | -<number> 
<number> ::= <non-zero-digit> | <non-zero-digit> <digit-sequence>